tests/std/tests/P0896R4_ranges_test_machinery/test.compile.pass.cpp

```diff
@@ -12,6 +13,10 @@ using namespace std;
 
 int main() {} // COMPILE-ONLY
 
+// GH-2358: <filesystem>: path's comparison operators are IF-NDR

```

Comment:
This file is for testing the ranges test machinery defined in `<range_algorithm_support.hpp>`. I think this test would fit better somewhere in `P0896R4_ranges_range_machinery`.